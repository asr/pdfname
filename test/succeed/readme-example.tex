\documentclass{article}
\usepackage{hyperref}

\hypersetup{%
  pdfauthor={Per Martin-LÃ¶f},%
  pdftitle={Introduction to the <TEX>\$\textbackslash lambda\$</TEX>-Calculus},%
  pdfcreationdate={D:20100409071401},%
}

\begin{document}
pdfname test.
\end{document}
