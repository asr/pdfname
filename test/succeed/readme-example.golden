ret > ExitSuccess
out > The full path name will be /tmp/martin-lof-2010.introduction-to-the-lambda-calculus.pdf
out >
